scripting/get5.sp

```diff
@@ -999,7 +1004,13 @@ public Action Timer_NextMatchMap(Handle timer) {
   int index = GetMapNumber();
   char map[PLATFORM_MAX_PATH];
   g_MapsToPlay.GetString(index, map, sizeof(map));
-  ChangeMap(map);
+
+  if (index > 1 || (g_DisplayGotvVeto.BoolValue && index == 1)) {
```

Comment:
Shouldn't this be ``== 0``? GetMapNumber is calculated as the sum of map scores, which is certainly 0 right after the veo.